tests/bluetooth/bsim/nrf_auraconfig/tester/src/nrf_auraconfig_test.c

```diff
@@ -0,0 +1,1169 @@
+/*
+ * Copyright (c) 2022-2023 Nordic Semiconductor ASA
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ctype.h>
+#include <stdlib.h>
+#include <strings.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/audio/pacs.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/bluetooth/audio/pbp.h>
+#include <zephyr/shell/shell.h>
+#include <bstests.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(test, CONFIG_TEST_LOG_LEVEL);
+
+#include "nac_test.h"
+
+#define SEM_TIMEOUT		    K_SECONDS(60)
+#define BROADCAST_ASSISTANT_TIMEOUT K_SECONDS(120) /* 2 minutes */
```

Comment:
```suggestion
#define BROADCAST_ASSISTANT_TIMEOUT K_SECONDS(120) /* 2 minutes */
``` Butter on lard as they say :)

Reply:
Will check if it is used and then remove it. We are not using the broadcast assistant feature in this test, remnants form the broadcast_sink sample from Zephyr